templates/channels/channel_claim.haml

```diff
@@ -182,6 +182,21 @@
                 -blocktrans
                   Easily add a two way number you have configured with Hub9 in Indonesia.
 
+    .claim-row.twiml-channel
+      .row
+        .span3
+          %a.btn.btn-primary.btn-primary-muted.btn-claim{href:"{% url 'channels.channel_claim_twiml_api' %}"}
+            .btn-container
+              .claim-glyph.icon-channel-twilio
+              %p.claim-text
+                -trans "TwiML API"
+
+        .span9
+          .claim-description
+            %div
+              -blocktrans
+                Connect to a TwiML API. You can use this to migrate away from Twilio and save on Twilio fees.
```

Comment:
"Connect to a service that speaks TwiML. You can use this to connect to TwiML compatible services outside of Twilio."